package.json

```diff
@@ -4,6 +4,8 @@
   "description": "Customizable Icons for React Native with support for NavBar/TabBar/ToolbarAndroid, image source and full styling.",
   "main": "dist/index.js",
   "bin": {
+    "add-font-assets": "./bin/add-font-assets.js",
+    "fa5-upgrade": "./bin/fa5-upgrade.sh",
```

Comment:
Does it make sense to prefix this somehow? I know `generate-icon` doesn't, but that doesn't mean it correct hehe.

Reply:
I don't know, looking through some ```.bin``` folders it doesn't seem conventional to have some common prefix.